src/PrestaShopBundle/Form/Admin/Type/TranslatableType.php

```diff
@@ -245,4 +277,24 @@ private function doesErrorFormAndCurrentFormMatches(FormInterface $errorForm, Fo
     {
         return $errorForm === $currentForm;
     }
+
+    /**
+     * Get default locale.
+     *
+     * @param array $locales
+     *
+     * @return array
+     */
+    private function getDefaultLocale(array $locales)
```

Comment:
Will this change create a behavior change on pages that are already migrated and use it ?

Reply:
@matks yes. This change brings this setting to life: https://prnt.sc/mulodb

If the mentioned setting is enabled, whenever you select a language in translatable input, it will memorize that language and will auto-select it in all translatable inputs all over back office.
It's a feature that was working on legacy pages but not implemented on migrated pages.

Reply:
Nice ! 👍